### ISSUE TITLE
New-Shortcut should have the ability to set Hotkey 

### ISSUE THREAD

**Issue by LMCO-psadt-devs** [PSAppDeployToolkit/PSAppDeployToolkit#270]
```
Param (
		. . .
		[Parameter(Mandatory=$false)]
		[ValidateNotNullorEmpty()]
		[string]$Hotkey,
		. . .
	)
	
$shortcut.Hotkey = $Hotkey
```

**Comment by Paradox355** [comment]
I concur.

**Comment by macxcool** [comment]
Instead of:
`$shortcut.Hotkey = $Hotkey` 
Maybe
`If ($Hotkey) {$shortcut.Hotkey = $Hotkey}`
would be better. +1 for this idea, though.

**Comment by seanlillis** [comment]
This will be included in the next release.

**Comment by seanlillis** [comment]
This is fixed in v3.7.0.
